modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/UnionPlannerTest.java

```diff
@@ -131,21 +51,38 @@ public void testUnion() throws Exception {
             "UNION " +
             "SELECT * FROM table3 ";
 
-        IgniteRel phys = physicalPlan(
-            sql,
-            publicSchema
-        );
-
-        System.out.println("+++ " + RelOptUtil.toString(phys));
-
-        assertNotNull(phys);
+        assertPlan(sql, publicSchema, isInstanceOf(IgniteReduceAggregateBase.class)
+            .and(hasChildThat(isInstanceOf(Union.class)
+                    .and(input(0, isTableScan(((TestTable)publicSchema.getTable("TABLE1")).name()))
+                    .and(input(1, isTableScan(((TestTable)publicSchema.getTable("TABLE2")).name())))
```

Comment:
Use `and` predicate after another `and` predicate (not after `input` predicate), this will align brackets and will be more readable.

Reply:
Fixed, thanks.